packages/ui/package.json

```diff
@@ -147,6 +147,7 @@
     "jsdom": "^25.0.1",
     "lint-staged": "^15.2.9",
     "react": "^17.0.2",
+    "react-router-dom": "^6.26.0",
```

Comment:
Can we please remove this altogether. We want to have compatible types, not using the package at all. Otherwise if we have to take on a new module that happens to use a different router altogether, we'll be back in the same position.

Reply:
We couldn't figure out a way to do that. The `types` we were using hard to mimic and kept giving compilation errors if removed.